### PR TITLE
RFC-0230 - Technical Editing

### PR DESCRIPTION
**Pull request by anselld**
## Description
Technical editing.

## Types of changes

* [ ] Bug fix (non-breaking change which fixes an issue)
* [ ] New feature (non-breaking change which adds functionality)
* [ ] Breaking change (fix or feature that would cause existing functionality to change)
* [ ] Feature refactor (No new feature or functional changes, but performance or technical debt improvements)
* [ ] New Tests
* [x] Documentation

## Checklist:

* [x] I'm merging against the `development` branch
* [ ] I ran `cargo-fmt --all` before pushing
* [ ] My change requires a change to the documentation.
* [ ] I have updated the documentation accordingly.
* [ ] I have added tests to cover my changes.

### CODE REVIEW

**Comment by anselld** [comment]
Hi @hansieodendaal, please could you review my changes - thanks. One comment : the link to 'transaction lock height' does not work. It is unclear what it is meant to link to.

File: RFC/src/RFC-0230_HTLC.md [hunk 1]
```diff
@@ -22,94 +22,94 @@ following conditions are met:
...
-This requires users constructing a transaction to:
-* MUST include a lock height in the kernel of their transaction,
-*  MUST include the lock height in the transaction signature to prevent lock height malleability.
+This means that users constructing a transaction:
+* MUST include a lock height in the kernel of their transaction; and
+* MUST include the lock height in the transaction signature to prevent lock height malleability.
 
-Tari Miners:
-* MUST not add any transaction to the mined [block] that has not already exceeded its lock height.
+Tari Miners [base node]s MUST NOT add any transaction to the mined [block] that has not already exceeded its lock height.
```
Comment (hansieodendaal) [inline RFC/src/RFC-0230_HTLC.md]:
```suggestion
Tari [base node]s MUST NOT add any transaction to the mined [block] that has not already exceeded its lock height.
```

File: RFC/src/RFC-0230_HTLC.md [hunk 2]
```diff
@@ -22,94 +22,94 @@ following conditions are met:
...
-process to construct a timelocked contract.
+Unlike Bitcoin, where this can be accomplished with a single transaction, in [Mimblewimble], HTLCs involve a multi-step
+process to construct a time-locked contract.
 
 The steps are as follows:
-* The sender MUST pay all the funds into a n-of-n [multisig] [UTXO].
-* All parties involved MUST construct a refund [transaction] paying back all funds to the sender, spending this n-of-n
+* The sender MUST pay all the funds into an n-of-n [multisig] [UTXO].
+* All parties involved MUST construct a refund [transaction] to pay back all funds to the sender who has spent this n-of-n
   [multisig] [UTXO]. However, this [transaction] has a [transaction lock height](#hashed-time-locked-contract) set in
```
[the remark this answers is not in the document]
Comment (anselld) [reply RFC/src/RFC-0230_HTLC.md]: Thanks.

**Comment by anselld** [comment]
Hi @hansieodendaal , I committed your suggestion - thanks.

**Comment by anselld** [comment]
Hi @hansieodendaal , the option to combine the commits on this PR is not there. Not sure what I'm doing wrong. I've combined the commits on the other PRs successfully.

**Pull request merged**